node/src/main/scala/org/bitcoins/node/networking/peer/PeerMessageReceiver.scala

```diff
@@ -60,9 +66,19 @@ class PeerMessageReceiver(
     * peer initialized a disconnection from us
     */
   private[networking] def initializeDisconnect(): PeerMessageReceiver = {
+    logger.debug(s"Initializing disconnect from $peer")
     state match {
-      case bad @ (_: Disconnected | _: InitializedDisconnectDone |
-          Preconnection) =>
+      case good @ (_: Disconnected) =>
```

Comment:
Case where we disconnected right before sending an intialize disconnect. In this case just don't try to reconnect.